lwt-unix/conduit_lwt_unix_ssl_real.ml

```diff
@@ -61,7 +60,7 @@ module Server = struct
       ?timeout sa cb =
     sa
     |> listen ~ctx ?backlog ?password ~certfile ~keyfile
-    |> Conduit_lwt_server.init ?stop (fun (fd, _) ->
+    >>= Conduit_lwt_server.init ?stop (fun (fd, _) ->
```

Comment:
I don't understand how conduit-ssl was compiling without that line. Are we testing this properly?

Reply:
Looking at travis, it seems like we don't test this variant with lwt ssl enabled. We should though